client/my-sites/themes/themes-selection.jsx

```diff
@@ -133,19 +136,22 @@ class ThemesSelection extends Component {
 		return options;
 	}
 
+	shouldShowPlaceholders() {
+		// 1 show placholders on initial load
+		// 2 show placeholders when we have themes to show and are fetching next pages
+		return this.props.isRequesting || ( this.props.themesCount === null && this.props.queryError === null );
```

Comment:
Hmm, with [this criterion](https://github.com/Automattic/wp-calypso/pull/12562/files#diff-da85ecea81ed7dcf86097e6d06009d88R61) in `shouldComponentUpdate`, `themesCount` here can only be `null` on first load, right?